nservicebus/pipeline/steps-stages-connectors_intro_core_[6,8).partial.md

```diff
@@ -0,0 +1,5 @@
+NServiceBus has the concept of a pipeline execution order that is executed when a message is received or dispatched. A *pipeline* refers to the series of actions taken when an incoming message is processed or an outgoing message is sent. This allows users to take full control of the incoming and outgoing message processing.
```

Comment:
Both this partial and the v8 version needs some rework since the text don't make sense to me, I'll work with Laila to propose something